dblookupext/historyRepository.go

```diff
@@ -481,6 +504,19 @@ func (hr *historyRepository) IsEnabled() bool {
 // RevertBlock will return the modification for the current block header
 func (hr *historyRepository) RevertBlock(blockHeader data.HeaderHandler, blockBody data.BodyHandler) error {
 	return hr.esdtSuppliesHandler.RevertChanges(blockHeader, blockBody)
+
+	// TODO(next PR): this function should contain the following code:
+	// err := hr.esdtSuppliesHandler.RevertChanges(blockHeader, blockBody)
+	// if err != nil {
+	//		return err
+	//	}
+	//
+	//	body, ok := blockBody.(*block.Body)
+	//	if !ok {
+	//		return fmt.Errorf("%w in historyRepository.RevertBlock for provided blockBody", errWrongTypeAssertion)
```

Comment:
will this trigger the retrial of the `RevertBlock` function? If so, it could harm the esdt supplies component

Reply:
should not happen. However, I've changed the order. If the rollback fails for other reasons (outport error or some other causes) what you have said will still happen.